docs/alternator/alternator.md

```diff
@@ -129,6 +129,28 @@ isolation policy for a specific table can be overridden by tagging the table
    Example: in order to query the contents of Scylla's system.large_rows,
    pass TableName='.scylla.alternator.system.large_rows' to a Query/Scan request.
 
+### Tag-based default time to live (TTL)
+ * Aside from the DynamoDB TTL mechanism based on the UpdateTimeToLive method,
+   alternator also exposes a way of defining default per-table time to live
+   parameter, expressed in seconds and accessible via a `system:default_time_to_live`
+   tag. The default time to live mechanism will automatically apply TTL
+   on all *new* updated and inserts, and remove them once their expiration period
```

Comment:
I think we be clearer here that the "TTL" in this mechanism is a relative time - after this number of seconds after the item is written, it gets deleted. This is different from DynamoDB's TTL feature which specifies absolute "expiration time", not a relative TTL.